app/Filament/Admin/Resources/WebhookResource/Pages/EditWebhookConfiguration.php

```diff
@@ -3,17 +3,38 @@
 namespace App\Filament\Admin\Resources\WebhookResource\Pages;
 
 use App\Filament\Admin\Resources\WebhookResource;
+use App\Filament\Admin\Widgets\DiscordPreview;
+use App\Models\WebhookConfiguration;
+use Filament\Actions\Action;
 use Filament\Actions\DeleteAction;
 use Filament\Resources\Pages\EditRecord;
+use Filament\Widgets\WidgetConfiguration;
 
 class EditWebhookConfiguration extends EditRecord
 {
     protected static string $resource = WebhookResource::class;
 
+    /**
+     * @return array<WidgetConfiguration>
+     */
+    protected function getHeaderWidgets(): array
+    {
+        return [
+            /* DiscordPreview::make([
+                'record' => $this->getRecord(),
+            ]), */
+        ];
+    }
+
     protected function getHeaderActions(): array
     {
         return [
             DeleteAction::make(),
+            Action::make('runNow')
+                ->label('Run now')
+                ->color('primary')
+                ->disabled(fn (WebhookConfiguration $webhookConfiguration) => count($webhookConfiguration->events) === 0)
+                ->action(fn (WebhookConfiguration $webhookConfiguration) => $webhookConfiguration->run()),
```

Comment:
This is supposed to be a test button, like the test email, right? If so, rename it from "run" to "test".